src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheck.java

```diff
@@ -56,7 +56,82 @@
  * <pre>
  * &lt;module name=&quot;IllegalCatch&quot;/&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * try {
+ *     // some code here
+ * } catch (Exception e) { // violation
+ *
+ * }
+ *
+ * try {
+ *     // some code here
+ * } catch (ArithmeticException e) {
```

Comment:
OK comment missing

Reply:
It will cause a violation since the next catch block is illegal, thats why didn't put the comment of `OK`

Reply:
Yes, but this catch block is ok, violation occurs on next one.